api/src/test/java/org/apache/iceberg/transforms/TestDatesProjection.java

```diff
@@ -94,6 +113,11 @@ public void testMonthStrictLowerBound() {
     assertProjectionStrict(spec, greaterThanOrEqual("date", date), Expression.Operation.GT, "2016-12");
     assertProjectionStrict(spec, notEqual("date", date), Expression.Operation.NOT_EQ, "2017-01");
     assertProjectionStrictValue(spec, equal("date", date), Expression.Operation.FALSE);
+
+    Integer date1 = (Integer) Literal.of("2017-12-02").to(TYPE).value();
```

Comment:
Nit: how about `anotherDate` instead of `date1`? It's strange to have both `date` and `date1` variables.